lib/connextclient/ConnextClient.ts

```diff
@@ -470,8 +470,17 @@ class ConnextClient extends SwapClient {
         case 'FAILED':
           return { state: PaymentState.Failed };
         default:
-          this.logger.debug(`no hashlock status for payment with hash ${rHash}: ${JSON.stringify(transferStatusResponse)}`);
-          return { state: PaymentState.Pending };
+          this.logger.debug(`no hashlock status for payment with hash ${rHash}: ${JSON.stringify(transferStatusResponse)} - attempting to reject app install`);
+          try {
+            await this.sendRequest('/reject-install', 'POST', {
+              appIdentityHash: transferStatusResponse.appIdentityHash,
+            });
+            return { state: PaymentState.Failed };
```

Comment:
Maybe log a line here right before returning indicating that we rejected the install and the payment status is failed?

Reply:
I've added a `debug` level log here and cleaned up the terminology in this function to be in line with connext.